backend/lib/models/User.js

```diff
@@ -27,12 +27,32 @@ userSchema.index({
   ownerId: 1,
   createdAt: -1,
 });
+
+userSchema.index(
+  {
+    name: "text",
+    firstName: "text",
+    lastName: "text",
+    about: "text",
+  },
+  {
+    weights: {
+      name: 2,
+      firstName: 2,
+      lastName: 2,
+      about: 1,
```

Comment:
Isn't `name` the aggregation of `firstName` and `lastName`? 
If so, using both at the same index might double their search weight.

Reply:
name is also used for organizations names... That might cause some conflict